tuf/keys.py

```diff
@@ -840,21 +843,23 @@ def verify_signature(key_dict, signature, data):
   if keytype == 'rsa':
     if _RSA_CRYPTO_LIBRARY == 'pycrypto':
```

Comment:
Issue, IMO:
 - While better than prev version, instruction to user fails to cover alternative of switching configuration to other library. Suggest something like "You could install PyCrypto (pip install PyCrypto or pip install tuf[tools]) or try switching your configuration (tuf/conf.py) to use pyca-cryptography if that is available instead)." Or, better yet, you could even issue that message specifically if another value *is* in _available_crypto_libraries! Something like (after checking _available_crypto_libaries!) "tuf/conf.py indicates that pycrypto should be used, but it is not available/installed. You could install it (pip install PyCrypto or pip install tuf[tools]), or you could change the value of RSA_CRYPTO_LIBRARY in tuf/conf.py to 'pyca-cryptography', which appears to be available on this system." (I also considered suggesting a setting "either" that would choose whichever is available, in priority order, but that could change if the preferred library were suddenly available due to some other installation. That seems vaguely risky. Might be worth considering anyway, since would be more hassle-free for user?)

Small issue, probably OK:
 - The structure here is a little redundant. Could be checking if _RSA_CRYPTO_LIBRARY is in supported crypto list and in _available_crypto_libraries. Still would need sloppy user instruction case, though, anyway. NBD.